examples/ios-objc/README.md

```diff
@@ -10,19 +10,29 @@ This is a sample iOS (Objective C) app designed to demonstrate how to use the Ru
 
 ### Step 1: Open project in Xcode
 
-- Open project path in Terminal & run `pod update`. Next open the `RudderTyperExample.xcworkspace`, located under `tests/e2e/ios-objc`, in Xcode.
+- Open project path in Terminal & run `pod update`. Next open the `RudderTyperExample.xcworkspace`, located under `examples/ios-objc`, in Xcode.
 
 ### Step 2: Configure RudderStack Properties
 
 - Update the `writeKey`, `dataPlaneUrl`, and `controlPlaneUrl` with your specific values in the `RudderConfig.plist`.
 
   ![RudderConfig](screenshots/rudderConfig_path.png)
 
-### Step 3: Add Generated Files
+### Step 3: Add/Regenerate Analytics Files
 
-- Navigate to the `tests/e2e/ios-objc` folder in the terminal.
-- Run `npx rudder-typer@next` to re-generate a client with the latest versions of these events.
-- These generated files will be located under the `generated` directory.
+- Navigate to the `examples/ios-objc` folder in the terminal.
+- Check the node version of the project & install the same by using following commands.
+
+```
+$ node --version
+
+$ node i
+
+$ node use
+```
```

Comment:
This has to be replaced with `nvm` (Node Version Manager). 
Moreover, this should not be part of the example app's instructions but rather the repo setup. It should be covered there.